### DISCUSSION TITLE
start the container with an external volume? [May 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2213] [Discord Threads]
> 🙋‍♂️ **jota**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### If you are using NC_DB - external volume is not required. However, you will have to configure s3 to store files. This could be done via our app store which is within (team and settings - left bottom)  
#  
> 🙋‍♂️ **jota**  
#### Can you give me a common example of which kind of file would be stored to s3?  
#  
> 🙋‍♂️ **jota**  
#### Is it necessary for internal NocoDB files or just in case I want to export something?  
#  
> 💁 **o1lab**  
#### the files u upload to attachment column  
#  
> 🙋‍♂️ **jota**  
#### Great  
#  
> 🙋‍♂️ **jota**  
#### Thanks a lot for the help. I'm trying NocoDB for my company and, if it works, I'll definitely become a larger sponsor (as a company; nowadays I'm a small sponsor using personal funds)  
#  
> 💁 **o1lab**  
#### thank you, really appreciate it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978342767732355082)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
